test/dash_analytics_test.dart

```diff
@@ -84,12 +84,6 @@ void main() {
     );
   });
 
-  tearDown(() {
```

Comment:
Removed this as each file system should only last a single test. if there are test failures because we're not deleting in tear down, this is actually a symptom that some other resource is getting leaked between tests.

Reply:
Good catch, I forgot to remove this after deciding to delete the directory at the top of the test instead. Seems like the tests all pass too